Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -171,7 +170,6 @@ void updateLayersForInteractionRegions(CALayer *layer, RemoteLayerTreeHost& host
             }
 
             configureLayerForInteractionRegion(interactionLayer.get(), makeString("WKInteractionRegion-"_s, region.elementIdentifier.toUInt64()));
-            [interactionLayer setCornerRadius:std::max(region.borderRadius, minimumBorderRadius)];
```

Comment:
we still need to call `setCornerRadius` here, but we can use `region.borderRadius` directly